cortex_xdr_client/api/scripts_api.py

```diff
@@ -133,6 +133,10 @@ def get_script_execution_results(self, action_id: int) -> Optional[GetScriptExec
         if "reply" not in resp_json:
             raise InvalidResponseException(response, ["reply"])
         reply = resp_json["reply"]
+        # Because pydantic does not support private attributes
+        if '_return_value' in reply['results'][0]:
```

Comment:
I guess this is only a 1 element array? Can we size check it first?

Reply:
Well yes, this also applies to other api endpoints like `endpoints_api`